--- rpi-firmware-old.mk	2018-06-05 07:46:35.000000000 +0200
+++ rpi-firmware.mk	2018-06-05 07:45:54.000000000 +0200
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-RPI_FIRMWARE_VERSION = 36f991382326907419f64917624427e59fd8e4a0
+RPI_FIRMWARE_VERSION = ce8652e2c743f02f04cb29f23611cbf13765483b
 RPI_FIRMWARE_SITE = $(call github,raspberrypi,firmware,$(RPI_FIRMWARE_VERSION))
 RPI_FIRMWARE_LICENSE = BSD-3-Clause
 RPI_FIRMWARE_LICENSE_FILES = boot/LICENCE.broadcom
